tests/model/test_model.py

```diff
@@ -2623,6 +2624,49 @@ def test__handle_reaction_event_for_msg_in_index(
     def update_message_flags_operation(self, request):
         return request.param
 
+    @pytest.mark.parametrize(
+        "event,initial_alerted_words, expected_alert_words",
+        [
+            case(
+                {"type": "alert_words", "alert_words": ["word1", "word2", "word3"]},
+                ["word1"],
+                ["word1", "word2", "word3"],
+                id="Add multiple alert words",
```

Comment:
minor, but to keep consistent: we tend to use underlines instead of spaces, in case of multiple parametrize.